catalog/solr/pom.xml

```diff
@@ -34,9 +34,6 @@
     </dependencyManagement>
     <modules>
         <module>catalog-solr-provider</module>
```

Comment:
❗️ We also need to remove those dependencies from the `catalog/solr/solr-app/pom.xml` file.